commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQL.java

```diff
@@ -514,15 +514,25 @@ public static PalantirSqlException handleInterruptions(long startTime,
             throw PalantirSqlException.create(cause);
         }
         String msg = "We got an execution exception that was an interrupt, most likely from someone " + //$NON-NLS-1$
-                "incorrectly ignoring an interrupt. "; //$NON-NLS-1$
+                "incorrectly ignoring an interrupt. Elaped time: {} \nError message: {} "
```

Comment:
We're putting the `Error message` etc into the message twice but not filling them in.  Looks wrong.  The optional logging of hte elapsed time messes this up and makes it tricky.  Could be worth just 2 big log sections for clarity?